src/encoder.rs

```diff
@@ -2261,10 +2263,12 @@ fn encode_partition_bottomup(seq: &Sequence, fi: &FrameInvariants, fs: &mut Fram
     // must_split overrides the minimum partition size when applicable
     let can_split = (bsize > fi.min_partition_size && is_square) || must_split;
 
-    let mut partition = PartitionType::PARTITION_NONE;
+    let mut partition = PartitionType::PARTITION_INVALID;
+    let mut best_partition = partition;
```

Comment:
Here and below with `PARTITION_NONE`, I think you can eliminate the `let mut partition...` lines and assign the values to `best_partition` directly.

Reply:
Done.